### PR TITLE
Merge to master

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: timer.js [hunk 1]
```diff
@@ -3,6 +3,40 @@ var timer = 135;
 var delayInMilliseconds = 1000;
 var interval = 0;
 var refresh = setInterval(timer_app, delayInMilliseconds);
+<<<<<<< HEAD
```
Comment (auscompgeek) [inline timer.js:6]: wot

**Comment by auscompgeek** [comment]
hound what are you doing

File: timer.js (hunk 1 above)
Comment (ghost) [reply timer.js:6]: What?

**Comment by auscompgeek** [comment]
Once you stop trying to push a merge conflict, fix your filenames, and ensure your JS doesn't simply outright error, I will review your code. Until then, please fix.

**Pull request closed without merging**